src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,373 @@
+    //static  Task[] tasks =new Task[100];
+
+    static final String BORDER="\n________________________________";
+    static final String BORDER_WITHOUT_SKIP="________________________________";
+    static final String FILE_PATH="data/duke.txt";
+    public static void main(String[] args) {
+        boolean byeDetected=false;
+
+        Scanner in = new Scanner(System.in);
+        String input;
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        String welcomeMessage = logo +
+                "________________________________\n"
+                +"Whats up people! duke.Duke is here to save the day\n"
+                +"Soooo, what can I do for you?\n"
+                +"________________________________\n";
+
+        String byeMessage =
+                "Aye captain. This is Duke Signing out!\n"
+                +"________________________________";
+
+        print(welcomeMessage);
+        try {
+            loadFileContents(FILE_PATH);
+        }catch (FileNotFoundException e){
+            print("Either File not found or File does not exist yet!");
+            createSavedFile();
+        }
+
+        while (!byeDetected) {
+            input = in.nextLine();
+            String[] words = input.split(" ");
+            switch (words[0].toLowerCase()) {
+            case "bye":
+                byeDetected = true;
+                break;
+
+            case "list":
+                listContent();
+                break;
+
+            case "done":
+                int taskNum = Integer.parseInt(words[1]);
+                tasks.get(taskNum - 1).maskAsDone();
+                print("Nice! I've marked this task as done:"
+                        + "\n"
+                        + "["
+                        + tasks.get(taskNum - 1).getStatusIcon()
+                        + "] "
+                        + tasks.get(taskNum - 1).getFullDescription()
+                        + BORDER
+                );
+                break;
+            case "delete":
+                int taskNumber = Integer.parseInt(words[1]);
+                if(listCount!=0){
+                    deleteItemFromList(taskNumber-1);
+                }
+                else{
+                    print("The list is empty! Nothing to delete!");
+                }
+
+                break;
+            case "todo":
+                try{
+                    ToDo t = validateToDo(words);
+                    addToList(t);
+                }catch (DukeException e){
+                    switch (e.exceptionType){
+                    case MISSING_DESCRIPTION:
+                        print(" The Description of a todo cannot be empty");
+                        break;
+                    }
+                }
+                printBorder();
+                break;
+
```

Comment:
There's inconsistency between where to have newline, before and after `break`. Maybe this should be standardised within the code?